### ISSUE TITLE
Same prediction for every molecules with pretrained and finetuned model. 

### ISSUE THREAD

**Issue by Changgun-Choi** [gmum/huggingmolecules#21]
Hello, 
By using quick_tour.py, results are always same for different molecules. I am confused about it. 
The result is also same for the fine-tuned model.  

![image](https://user-images.githubusercontent.com/72928938/151857615-a69e2478-f801-4a94-ad2f-9d620f80617e.png)

**Comment by Changgun-Choi** [comment]
Hello, 
I tried with it several times but I can't understand why the prediction is the same for all molecules. 
I would really appreciate it if you help me. 

Best regards,

**Comment by panpiort8** [comment]
Hello :) could you provide the rest of your code, please? Could you also explain how you run your code? Line by line from command-line, or with Jupyter notebook? Unfortunately, I am unable to deduce it from your screen :)

**Comment by Changgun-Choi** [comment]
![image](https://user-images.githubusercontent.com/72928938/152409520-0e07dcdf-9c49-4182-88ea-71b04a3b3b4a.png)

Hello! thank you very much for the reply. 
 I run the whole quick.tour.py without changing anything. So it downloads the dataset and train and predicts 2 different molecules with the same result. 

I expected featurizer creates different results but always same as you can see here. 
![image](https://user-images.githubusercontent.com/72928938/152412227-33ef61fb-0190-45f7-9906-dca4e6be0e17.png)

**Comment by panpiort8** [comment]
Hmm, it's quite strange. Could you show me your conda environment, please? Moreover, could you try to reproduce this bug from a freshly cloned huggingmolecules repository? Cause I see you've modified the `src/` directory - maybe the modification introduced some bug? Probably not, but I cannot see many other options.

Btw: `quick_tour.py` imports `MatFeaturizer` from the version of huggingmolecules that is currently installed in your python environment (`from huggingmolecules import MatModel, MatFeaturizer`). If you modify the code, you should also update the installed huggingmolecules package by running `pip install -e src/`.

**Comment by Changgun-Choi** [comment]
Thank you for the reply!

I cloned the new git repository but I don't see any difference. So, this bug doesn't appear to you and gives different predictions? 

In terms of 'pip install -e src/.', it gives an error below so I downloaded pytorch myself.
 
![image](https://user-images.githubusercontent.com/72928938/152539072-e2e8d1ce-6188-4ead-b558-17262748063c.png)
![image](https://user-images.githubusercontent.com/72928938/152540540-e9e22b77-5716-4005-ba80-f7bbf6e52408.png)

![image](https://user-images.githubusercontent.com/72928938/152541226-46dce6ac-5073-4805-b1b3-e445bc92c8fd.png)

**Comment by panpiort8** [comment]
Yep, unfortunately, I cannot reproduce the bug :< I've tried on three different machines, but everything works. But looking at the above screenshots, I'm not sure if you tried to reinstall huggingmolecules once again, after installing pytorch 1.7.0. Could you please run `pip install -e src/` again?  Maybe I am wrong, but the only possibility, I see at the moment is that you have still some bugged version of huggingmolecules installed in your conda environment.

Btw: in branch `change_setup_requirements`,  I've updated torch requirements to match the versions available with pip. If you use this branch to run `pip install -e src/`, it should reinstall the package successfully.

**Comment by Changgun-Choi** [comment]
Hello, I created a new conda environment and started from the beginning. 
I ran 'pip install -e src/' and 'requirements.txt' but I get these dependencies issues. I think those should be downloaded before running the script. 
Did you also try creating a new conda environment when you couldn't reproduce the bug? 
Thank you!

![image](https://user-images.githubusercontent.com/72928938/152677598-e543d7b8-fa17-427b-833c-44e36c8be809.png)
